doc/guides/debugging/host-tools.rst

```diff
@@ -54,6 +54,7 @@ These debug host tools are compatible with the following debug probes:
 
 - :ref:`opensda-daplink-onboard-debug-probe`
 - :ref:`jlink-external-debug-probe`
+- :ref:`stlink-v21-onboard-debug-probe`
```

Comment:
Do you need to add a similar line to the J-Link host tools section?

Reply:
Yeah, will do